### PR TITLE
nitrogfx NCGR

### PR DESCRIPTION
**Pull request by red031000**
still need to do writing

### CODE REVIEW

**Comment by red031000** [comment]
ready for review

File: tools/msgenc/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+msgenc
```
Comment (PikalaxALT) [inline tools/msgenc/.gitignore:1]: Missing newline at EOF

File: tools/nitrogfx/gfx.c [hunk 2]
```diff
@@ -297,6 +359,98 @@ void WriteImage(char *path, int numTiles, int bitDepth, int metatileWidth, int m
...
+        case 8:
+            ConvertToTiles8Bpp(image->pixels, pixelBuffer, numTiles, metatilesWide, metatileWidth, metatileHeight, invertColors);
+            break;
+    }
+
+    WriteGenericNtrHeader(fp, "RGCN", bufferSize + 0x20, byteOrder);
+
+    unsigned char charHeader[0x20] = { 0x52, 0x41, 0x48, 0x43, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                       0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                       0x00, 0x00, 0x18, 0x00, 0x00, 0x00 };
```
Comment (PikalaxALT) [inline tools/nitrogfx/gfx.c:414]: Line breaks after 13 bytes instead of 16 bytes -- why??

Comment (red031000) [reply tools/nitrogfx/gfx.c:414]: 120 character soft limit

**Comment by red031000** [comment]
addressed

**Pull request merged**